### PR TITLE
feat(vue-demo, react-demo): remote debug, add remote bundleURL entry

### PR DESCRIPTION
**Pull request by shenchaoran**
Before submitting a new pull request, please make sure:

- [ ] Test cases have been added/updated/passed for the code you will submit.
- [x] Documentation has added or updated.
- [x] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters.
- [x] Squash the repeat code commits, short patches are welcome.

### CODE REVIEW

File: examples/hippy-vue-demo/src/pages/debug.vue [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+.demo-input .input {
+  width: 300px;
+  height: 48px;
+  color: #242424;
+  border-width: 1px;
+  border-color: #ccc;
+  font-size: 16px;
+  margin: 20px;
+  placeholder-text-color: #aaa;
+  /* underline-color-android: #40b883; */
```
Comment (zoomchan-cxj) [inline examples/hippy-vue-demo/src/pages/debug.vue:110]: 多写了？

File: examples/hippy-vue-demo/src/pages/remote-debug.vue [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+.demo-remote-input .remote-input {
+  width: 350px;
+  height: 80px;
+  color: #242424;
+  border-width: 1px;
+  border-color: #ccc;
+  font-size: 16px;
+  margin: 20px;
+  placeholder-text-color: #aaa;
+  /* underline-color-android: #40b883; */
```
Comment (zoomchan-cxj) [inline examples/hippy-vue-demo/src/pages/remote-debug.vue:122]: /* underline-color-android: #40b883; */ 去除

Comment (shenchaoran) [reply examples/hippy-vue-demo/src/pages/remote-debug.vue]: removed

File: examples/hippy-vue-demo/src/pages/debug.vue (hunk 1 above)
Comment (shenchaoran) [reply examples/hippy-vue-demo/src/pages/debug.vue]: removed

File: docs/guide/debug.md [hunk 3]
```diff
@@ -363,3 +371,100 @@ webpack(webpackConfig, (err, stats) => {
...
+
+<img src="../assets/img/remote-debug-demo.gif" width="80%" />
+
+具体配置流程如下：
+
+1. 前端接入：
+
+   - 安装新一代调试工具： `npm i -D @hippy/debug-server-next@latest`
+
+   - 修改 Webpack 配置，添加远程调试地址（默认为本地），`remote` 字段改为远程调试地址，你需要自己[私有化部署远程调试服务](https://github.com/hippy-contrib/debug-server-next/blob/main/doc/deploy.md)
```
[the remark this answers is not in the document]
Comment (shenchaoran) [reply docs/guide/debug.md]: 已补充~

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**